include/ua_client_highlevel.h

```diff
@@ -212,6 +212,67 @@ UA_Client_readUserExecutableAttribute(UA_Client *client, const UA_NodeId nodeId,
                                      &UA_TYPES[UA_TYPES_BOOLEAN]);
 }
 
+/**
+ * Historical Access
+ * ^^^^^^^^^^^^^^^^^
+ * The following functions can be used to read a single node historically.
+ * Use the regular service to read several nodes at once.
+ */
+typedef UA_Boolean
+(*UA_HistoricalIteratorCallback)(const UA_NodeId nodeId, const UA_Boolean isInverse,
+                                 const UA_Boolean moreDataAvailable,
+                                 const UA_HistoryData *data, void *handle);
+
+/* Don't call this function, use the typed versions */
+UA_HistoryReadResponse UA_EXPORT
+__UA_Client_readHistorical(UA_Client *client, const UA_NodeId nodeId,
```

Comment:
I don't see any code using this "hidden" function. E.g. in a static inline method.
Do we need the __ at the beginning?

Rename to `...historyRead`